client/client.go

```diff
@@ -269,15 +329,34 @@ func (c *Client) SearchWord(directory, word string) ([]string, error) {
 		return nil, err
 	}
 
-	trapdoors := dirInfo.indexer.ComputeTrapdoors(word)
-	documents, err := c.searchCli.SearchWord(context.TODO(), sserver1.SearchWordArg{TlfID: dirInfo.tlfID, Trapdoors: trapdoors})
+	keyGens, err := c.searchCli.GetKeyGens(context.TODO(), dirInfo.tlfID)
```

Comment:
We should add a TODO to consider caching this, since it adds a round trip time for every search and will change very rarely.  The server could tell us when we do a search without all the necessary keygen trapdoors.